### ISSUE TITLE
Surface constructors are inconsistent

### ISSUE THREAD

**Issue by mike-colls** [mikebmcl/P0267_RefImpl#57]
I have been experimenting with this library and have a question about the following constructors.

`image_surface image{io2d::format::argb32, 640, 480};`
`display_surface display_surface{ 640, 480, io2d::format::argb32 }`

Is there a reason why the argument order for these two constructors are not the consistent?

Many thanks for work on this library and standards proposal. My efforts using this library for potting are here https://github.com/mike-colls/p0267_plot

**Comment by hatcat** [comment]
Not really. We are sprucing everything up post-Toronto to make the implementation compliant with R6 of the paper, and then we'll probably go through and resolve any inconsistencies like this before addressing further issues. Thanks for your input!
